examples/basics/src/components/Welcome.astro

```diff
@@ -0,0 +1,200 @@
+---
+import astroLogo from '../assets/astro.svg';
+---
+
+<div id="container">
+	<main>
+		<section id="hero">
+			<a href="https://astro.build"
+				><img src={astroLogo.src} width="115" height="48" alt="Astro Logo" /></a
+			>
```

Comment:
The link is currently taking the whole available width, which can lead to accidental clicks.
![image](https://github.com/user-attachments/assets/84d424cd-6778-48af-aeaa-ef50a62c9642)

I guess the easiest way to fix would be just to add:
```css
#hero > a {
	width: fit-content;
}
```

Additionally, this link can be more friendly to screen readers. Currently, it reads out as "Astro logo, graphic, visited, link", the recommendation is that when [images serve a function rather than convey information](https://www.w3.org/WAI/tutorials/images/functional/), we want to indicate the action vs. a description of the image.

```suggestion
			<a href="https://astro.build"
				><img src={astroLogo.src} width="115" height="48" alt="Astro Homepage" /></a
			>
```

Reply:
Any reason why? I had it as `align-items: start` at first

Reply:
In general I find it’s preferable to work with alignments over widths. It’s not a huge deal here where refactoring the component etc. is not super likely, but working with how stuff flows (with widths therefore determined implicitly) is usually more robust than setting sizes explicitly, which are more likely to then cause issues with overflow etc. (This is totally more a comment about a conceptual/systematic way of thinking rather than a “this code doesn’t work” comment.)

Setting `align-self: start` on the link would also meet this way of thinking.